clang/lib/Analysis/FlowSensitive/Models/UncheckedOptionalAccessModel.cpp

```diff
@@ -599,7 +599,7 @@ void transferAssignment(const CXXOperatorCallExpr *E, BoolValue &HasValueVal,
                         LatticeTransferState &State) {
   assert(E->getNumArgs() > 0);
 
-  if (auto *Loc = cast<RecordStorageLocation>(
+  if (auto *Loc = dyn_cast_or_null<RecordStorageLocation>(
```

Comment:
If getStorageLocation may return nullptr, it should be cast_or_null. `dyn_cast` means it may be not RecordStorageLocation

Reply:
updated